### PR TITLE
Helpers: Fix faulty timeago helper.

### PR DESCRIPTION
**Pull request by jagtalon**
Two things that's faulty with the `timeago` helper:

1. `.replace` method should be a regex in the first parameter and not a string.
2. We can't have implicit variables in the handlebars helpers. ~~Perl idioms seem to have seeped into our JavaScript! `0` isn't the same as `false`. `0 ? 1 : 2` will always result to `2`.~~

Please review, @MariagraziaAlastra.

### CODE REVIEW

File: src/js/ia/Helpers.js [hunk 1]
```diff
@@ -30,7 +30,7 @@
         var timestring = full? " days ago" : "d";
         if (date) {
             // expected date format: YYYY-MM-DDTHH:mm:ssZ e.g. 2011-04-22T13:33:48Z
-            date = date.replace("/T.*Z/", "");
+            date = date.replace(/T.*Z/, "");
```
Comment (jdorweiler) [inline src/js/ia/Helpers.js]: Can momentjs use the date string as-is?  I found this in the docs http://momentjs.com/docs/#/parsing/special-formats/

Comment (jagtalon) [inline src/js/ia/Helpers.js]: @jdorweiler Yes I was thinking that too! I think this is a relic of _before_ we used momentjs. I could change it to that instead if we're not in a hurry

File: src/templates/dev_pipeline.handlebars [hunk 2]
```diff
@@ -142,12 +142,12 @@
                       {{#exists pr 'issue_id'}}
                         {{#eq pr.status 'open'}}
                               <a href="https://github.com/duckduckgo/zeroclickinfo-{{repo}}/issues/{{pr.issue_id}}" class="one-line" id="pr" title="{{format_time last_update}}">
-                                {{timeago last_update 0}}
```
Comment (MariagraziaAlastra) [inline src/templates/dev_pipeline.handlebars]: @jagtalon You're not even supposed to use 0 here. It should just have no arguments. 
Options are the elements to render after the helper, and they never get passed, so it's not a problem to not pass a value for the from_beta value. Look at the overview and issues handlebars.

File: src/templates/dev_pipeline.handlebars [hunk 3]
```diff
@@ -142,12 +142,12 @@
...
+                                {{timeago last_update false false}}
```
Comment (MariagraziaAlastra) [inline src/templates/dev_pipeline.handlebars]: false false? What? See previous comment

File: src/js/ia/Helpers.js (hunk 1 above)
Comment (MariagraziaAlastra) [inline src/js/ia/Helpers.js]: @jagtalon @jdorweiler The reason why we're stripping the hours, minutes and seconds is that this way it's easier to get an int number (just days) as difference, when calculating the elapsed time.
There's no reason to change this part.

**Comment by MariagraziaAlastra** [comment]
@jagtalon That "from_beta" variable is a recent addition to our code. It's supposed to be null (aka you shouldn't pass it at all) when the timestamp doesn't come from the beta server.
It was a way to just change the code inside the overview handlebars in the "Recently on beta" section without having to change it everywhere else as well.

**Comment by jagtalon** [comment]
@MariagraziaAlastra oh ok lemme try it! Maybe I misunderstood

**Comment by jagtalon** [comment]
Hey, @MariagraziaAlastra! I tested it, and I've verified that we have to have explicit arguments, e.g., `{{timeago last_update false false}}` instead of `{{timeago last_update}}` only, because an object is always passed as a parameter. Here's a quick demo (you can open [this page](https://s.codepen.io/jagtalon/debug/66536f6334d0aed68bce47559d1634a3) which has HandlebarsJS loaded, and you can type these in the console):

screen shot 2015-12-16 at 10 35 05 am

So in the first line I created a helper function that accepts one argument. I invoke that function using `{{test}}` and you'll see that the parameter `hi` is not `null`, but rather an object that is always passed as the last argument. So in our case, running `{{timeago last_update}}` on its own messes up our logic in the Handlebars helper function which causes the wrong dates to come up:

screen shot 2015-12-16 at 10 42 07 am

I hope that explains the use of `{{timeago last_update false false}}`!

**Comment by MariagraziaAlastra** [comment]
LGTM, thanks @jagtalon

**Pull request merged**